_opensearch/date_math_support.md

```diff
@@ -0,0 +1,61 @@
+---
+layout: default
+title: Date math support for index names
+nav_order: 92
+---
+
+# Date math support for index names
+
+Date math is shorthand arithmetic for finding relative dates.
+
+If you're indexing time-series data with the dates mapped in the index names, you can use date math in your queries to filter index names and limit the number of searched indices.
+
+## Date math syntax
+
+The date math syntax for an index name is as follows:
+
+```
+<static_name{date_math_expr{date_format|time_zone}}>
+```
+
+where:
+
+- `static_name` is the unchanged or static portion of the index name. To use the characters `{` and `}` in the static part of an index name, escape them with a backslash `\`.
```

Comment:
Let's keep the same pattern for all the bullet points. So either `some-thing is` or `some-thing:`.